recipe-server/client/control_new/components/recipes/Details.js

```diff
@@ -0,0 +1,45 @@
+import { Card } from 'antd';
+import { Map } from 'immutable';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+
+export default function Details({ recipe }) {
+  return (
+    <div className="recipe-details">
+      <Card key="recipe-details" title="Recipe">
```

Comment:
I couldn't for the life of me figure out how to do this. And this doesn't seem to work either.